vllm/model_executor/layers/quantization/deepspeedfp.py

```diff
@@ -128,8 +138,18 @@ def apply(self,
               x: torch.Tensor,
               bias: Optional[torch.Tensor] = None) -> torch.Tensor:
         weight = layer.weight
-        y = weight.ds_dequantize()
-        return F.linear(x, y, bias)
+        if self.enable_fused_kernel:
+            y = g_matmul_fp8(
+                x, 
+                weight, 
+                weight.quantization_scales(), 
+                weight.fp_quantizer.group_size
+            )
+            return y if bias is None else (y + bias)
```

Comment:
```suggestion
            if bias is not None:
                y += bias
            return y
```
avoids creating a new tensor.